### ISSUE TITLE
Exporting theme to wrong folder path deleted all content in said folder

### ISSUE THREAD

**Issue by haelog** [themix-project/oomox#355]
Marking this issue here as I haven't found any other mention of it. I accidentally exported a theme to my /Downloads folder and once I opened it, the theme files was all there was inside it. I don't know how that's possible, if it's done by intent or was a bizarre bug but all the files are gone definitely and I'm lucky it wasn't anything crucial. OS: Manjaro/KDE Plasma.

**Comment by actionless** [comment]
yes, that's done intentionally, so you could adjust the exported theme-dir name (such feature was requested multiple times before)

everytime you open export dialog, export path defaults to parent dir of previously exported theme + '/' + name of the current theme
